lib/src/modules/conditional.dart

```diff
@@ -30,9 +32,33 @@ abstract class _Always extends Module with CustomSystemVerilog {
 
   final Uniquifier _portUniquifier = Uniquifier();
 
-  _Always(this.conditionals, {super.name = 'always'}) {
+  _Always(this.conditionals,
```

Comment:
We should add a doc comment for the constructor of  `_Always` explaining the functionality of the reset-related stuff.